### PR TITLE
adding Responsive Background colors with opacity

### PR DESCRIPTION
**Pull request by Charan42**
adding new CSS classes for Responsive Background colours with opacity

### CODE REVIEW

**Comment by Charan42** [comment]
[Frontpage banners](https://trello.com/c/wMV8A2RM/262-frontpage-banners)

File: src/shared/base/colors.css [hunk 1]
```diff
@@ -127,7 +127,27 @@
   background-color: var(--hw-color-opaque-black-20);
 }
 
+/* Responsive Background colors with opacity*/
 
-
-
-
+.hw-background-color-opaque-primary-mobile{
```
Comment (viggo) [inline src/shared/base/colors.css]: "mobile" sets background for desktop? Please fix

File: src/shared/base/colors.css [hunk 2]
```diff
@@ -127,7 +127,27 @@
...
+  @media(--large){
+    background-color: var(--hw-color-opaque-primary);
+  }
+}
+.hw-background-color-opaque-primary-desktop{
```
Comment (viggo) [inline src/shared/base/colors.css]: "desktop" sets background for mobile? Please fix

File: src/shared/base/colors.css [hunk 3]
```diff
@@ -127,7 +127,27 @@
...
+  @media(--max-medium),
+  @media(--high-res-mobile){
+   background-color: var(--hw-color-opaque-primary);
+  }
+}
+.hw-background-color-opaque-black-mobile{
```
Comment (viggo) [inline src/shared/base/colors.css]: "mobile" sets background for desktop? Please fix

File: src/shared/base/colors.css [hunk 4]
```diff
@@ -127,7 +127,27 @@
...
+  @media(--large){
+    background-color: var(--hw-color-opaque-black-20);
+  }
+}
+.hw-background-color-opaque-black-desktop{
```
Comment (viggo) [inline src/shared/base/colors.css]: "desktop" sets background for mobile? Please fix

**Pull request merged**